README.md

```diff
@@ -144,7 +144,9 @@ manualDismissTrigger.trigger() // Dismiss
 - Set your custom view on the `UINotificationCenter`:
 
 ```swift
-UINotificationCenter.current.defaultNotificationViewType = MyCustomNotificationView.self
+UINotificationCenter.current.configuration = UINotificationCenterConfiguration(
```

Comment:
This is the breaking change. I've decided to introduce this to reduce the number of mutations inside `UINotificationCenter`. I've updated the documentation accordingly.

Reply:
Out of scope for this pull request, but given this breaking change, would we want to consider stepping away from the `UI`-prefix to prevent confusion between this type and UIKit-types before bumping to a new major version?

Reply:
I don't think we should want that. By removing the UI-prefix we will actually match [the foundation `Notification`](https://developer.apple.com/documentation/foundation/notification) type and we take away the indication that we're talking about User Interface notifications. Thoughts?

Reply:
It'd be more about removing the confusion this is a UIKit-type, and possible naming overlap in the future. So it'd be more like `WTUINotification`, for example. Or we could step away to naming like `Toast`. But certainly a separate discussion :)